feat(doc): doc updates (#3164)
scaleway/docs-content: dedibox/vps/how-to/manage-vps.mdx

---
meta:
  title: How to manage a Dedibox VPS
  description: This page explains how to manage a Dedibox VPS
content:
  h1: How to manage a Dedibox VPS
  paragraph: This page explains how to manage a Dedibox VPS
tags: vps dedibox manage
dates:
  validation: 2024-05-15
  posted: 2024-05-15
categories:
  - dedibox-vps
---

You can manage your Dedibox VPS directly from the Dedibox console.

<Macro id="requirements" />

- A Dedibox account logged into the [console](https://console.online.net)
- [Created a Dedibox VPS](/dedibox/vps/how-to/create-vps/)

## How to manage a Dedibox VPS from its information dashboard

> 1. Open the Dedibox console home page and click **Dedibox VPS** in the navigation at the top of the page. This will bring up a list of your virtual machines. 

Reviewer: ```suggestion
1. Log in to your Dedibox console account and click **Dedibox VPS** in the navigation at the top of the page. This will bring up a list of your Dedibox VPS. 
```

        <Message type="tip">
            You can filter your Dedibox VPS by offer type using the drop-down list.
        </Message>

> 2. Choose the Dedibox VPS you wish to manage and click Manage. This action leads you to the VPS information dashboard.

Reviewer 2: ```suggestion
2. Choose the Dedibox VPS you wish to manage and click **Manage**. This action leads you to the VPS information dashboard.
```

        <Message type="tip">
            Alternatively, you can directly access the VPS information dashboard by
	        clicking on the VPS Hostname or IP address.
        </Message>
 
### How to view the root password of a Dedibox VPS

> 1. Access the VPS information dashboard and click the **view icon** (eye symbol) next to Password.

Reviewer: ```suggestion
1. Access the VPS information dashboard and click the **view icon** (eye symbol) next to **Password**.
```

> 2. The password of your Dedibox VPS displays. You can copy and paste it in your SSH client during connection to your VPS.

Reviewer 2: ```suggestion
2. The password of your Dedibox VPS displays. You can copy and paste it into your SSH client during connection to your VPS.
```

    <Message type="tip">
        Click the view icon again to hide the password.
    </Message>

### How to update the root password of a Dedibox VPS

1. Access the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) and click **Edit Password**.

> 2. Input your new password along with its confirmation, then click **Confirm** to finalize the update. You will receive a confirmation notification indicating the successful password change.

Reviewer: ```suggestion
2. Type your new password along with its confirmation, then click **Confirm** to finalize the update. You will receive a confirmation notification indicating your password was changed.
```

> 3. Reboot your VPS to ensure the changes take effect. After this, you can manage your VPS using the new password.

Reviewer: ```suggestion
3. Reboot your VPS to ensure the changes are taken into effect. After this, you can manage your VPS using the new password.
```

### How to update the hostname of a Dedibox VPS

1. Click the **Edit icon** on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) next to **Hostname**.
2. Enter the new hostname for your VPS and confirm by clicking **Confirm**. A notification will confirm the update.

### How to reboot a Dedibox VPS

> 1. Click Reboot on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) to reboot your Dedibox VPS.

Reviewer 2: ```suggestion
1. Click **Reboot** on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) to reboot your Dedibox VPS.
```

> 2. Confirm the action by clicking OK in the pop-up prompt. Your VPS will proceed to reboot.

Reviewer 2: ```suggestion
2. Confirm the action by clicking **OK** in the pop-up prompt. Your VPS will proceed to reboot.
```

### How to stop a Dedibox VPS

> 1. Click Stop on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard), if you wish to stop your Dedibox VPS.

Reviewer 2: ```suggestion
1. Click **Stop** on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard), if you wish to stop your Dedibox VPS.
```

2. Confirm the action by clicking **OK** in the pop-up prompt. Your VPS will begin the stopping process.

### How to start a Dedibox VPS

> 1. Click Start on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) to start your Dedibox VPS.

Reviewer 2: ```suggestion
1. Click **Start** on the [VPS information dashboard](#how-to-manage-a-dedibox-vps-from-its-information-dashboard) to start your Dedibox VPS.
```
